console/application.go

```diff
@@ -50,6 +52,10 @@ func NewApplication(name, usage, usageText, version string, useArtisan bool) con
 func (r *Application) Register(commands []console.Command) {
 	for _, item := range commands {
 		item := item
+		arguments, err := argumentsToCliArgs(item.Extend().Arguments)
+		if err != nil {
+			color.Errorln(errors.ConsoleCommandRegistrationFailed.Args(item.Signature(), err).Error())
```

Comment:
After printing the error here, it might be better to use `continue` to skip the iteration, right?

Reply:
added `continue`

I did it without `continue` because for me it provides at least some context for user 
```
./artisan smtp:add  --help      
  ERROR   Registration of command 'smtp:add' failed: required argument 'host' should be placed before any not-required arguments
Description:
   Add SMTP server

   Stores SMTP server by provided server_id into database


Usage:
   artisan [global options] smtp:add [options]
```

with `continue` 
```
./artisan smtp:add  --help
  ERROR   Registration of command 'smtp:add' failed: required argument 'host' should be placed before any not-required arguments
  ERROR   Command 'smtp:add' is not defined. Did you mean one of these?
```